Sources/SQLKit/Query/SQLColumnConstraintAlgorithm.swift

```diff
@@ -92,12 +92,20 @@ public enum SQLColumnConstraintAlgorithm: SQLExpression {
     public func serialize(to serializer: inout SQLSerializer) {
         switch self {
         case .primaryKey(let autoIncrement):
+            if autoIncrement {
+                if serializer.database.dialect.supportsAutoIncrementUsingDefaultFunction{
```

Comment:
What do you think about this structure:
```swift
if serializer.database.dialect.supportsAutoIncrement {
    if let function = serializer.database.dialect.autoIncrementFunction {
        // serialize DEFAULT + function
    } else {
        // serialize autoIncrementClause
    }
} else {
    // warn not supported 
}
```

Reply:
Will probably make the code cleaner, so I’ll try to make it so.